engines/producers/app/controllers/producers/api/v1/producers_controller.rb

```diff
@@ -0,0 +1,63 @@
+module Producers
+  module Api
+    module V1
+      class ProducersController < ApplicationController
+        before_action :load_producer, only: [:show, :update, :destroy]
+
+        # GET /api/v1/producers
+        #
+        def index
+          # no op
```

Comment:
Maybe we could respond with the providers that are suppliers in the groups of the user. Too much? :joy: